### ISSUE TITLE
function with comment after the "def" does not appear in outline list

### ISSUE THREAD

**Issue by gittar** [Hirse/brackets-outline-list#99]
**Details about your environment**

* **Brackets Version:** 
1.10 experimental build
* **Brackets Outline List Version:** 
1.4

**What did you do? Please include the actual source code causing the issue.**
```python
def plot1(ax, X=None, C=None):
    pass

def plot2(ax, X=None, C=None): #some comment
    pass
```

**What did you expect to happen? How did you expect the Outline to look?**
Expected: `plot1` and `plot2` appear in outline list

**What actually happened? How did the Outline actually look?**
Observed: only `plot1` appeared in the outline list

Likely reason: The comment (which is valid python)

**Comment by Hirse** [comment]
Yes, the comment seems to be the issue here.

**Comment by Hirse** [comment]
Fixed in #102, should be in the next update.
